Makefile.PL

```diff
@@ -351,7 +351,7 @@ if (-d 'inc/patches') {
     if (which('patch')) {
         for my $file (glob 'inc/patches/*.patch') {
             # replacing \n with \n will change \n to \r\n on Win, without this patch.exe will crash
-            system "cd \Q$mxbase\E && perl -pe \"s/\\n/\\n/\" ../../\Q$file\E | patch -p1 -N";
+            system "cd $mxbase && perl -pe \"s/\\n/\\n/\" ../../$file | patch -p1 -N";
```

Comment:
> $file is one of the file in 'inc/patches'.

And if that file in the future will contains spaces or ; or anything else - part of file name will be incorrectly injected as part of shell command.

I agree there is no security issue right now, but it's just a mandatory rule: everywhere you run shell command or eval string make sure everything is properly escaped. It's much easier to add these redundant (ATM) regexp checks and never think about this issue again - no matter now we'll rename or auto-detect paths for patches and mxbase.